shared/reducers/wallets.js

```diff
@@ -18,6 +18,10 @@ export default function(state: Types.State = initialState, action: WalletsGen.Ac
       return state.setIn(['assetsMap', action.payload.accountID], I.List(action.payload.assets))
     case WalletsGen.paymentsReceived:
       return state.setIn(['paymentsMap', action.payload.accountID], I.List(action.payload.payments))
+    case WalletsGen.secretKeyReceived:
+      return state.setIn(['secretKeyMap', action.payload.accountID], action.payload.secretKey)
+    case WalletsGen.secretKeySeen:
+      return state.set('secretKeyMap', I.Map())
```

Comment:
why have this be a map and set it based on account but clear the entire map? likely this can just be one value right

Reply:
if you switch accounts it can just clear it